<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pulse Analyzer</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="centralWidgetContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>961</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>961</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="controlsPanelWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>331</width>
           <height>611</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_load_ref">
            <property name="text">
             <string>Load Reference signal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_ref_file">
            <property name="text">
             <string>Reference: Not loaded</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_load_readings">
            <property name="text">
             <string>Load Readings Signal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_readings_file">
            <property name="text">
             <string>Readings: not loaded</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Reference Column Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ref_col">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Readings Column Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_readings_col">
            <property name="text">
             <string>Readings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Time Column Name (Optional) :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_time_col">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Similarity Threshold (0.0-1.0) :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="dsp_threshold">
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>0.700000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_analyze">
            <property name="text">
             <string>Analyze</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_status">
            <property name="text">
             <string>Status: Ready</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="plotsAreaWidget" native="true">
        <widget class="QWidget" name="plot_widget_placeholder_raw" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>330</y>
           <width>581</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="plot_widget_placeholder_filtered" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>581</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
